bin/run-ted.sh

```diff
@@ -6,8 +6,22 @@ if [ ! -f DeepSpeech.py ]; then
     exit 1
 fi;
 
+if [ ! -d "${ds_dataroot}" ]; then
+    ds_dataroot = "data"
+fi;
+
+# Warn if we can't find the train files
+if [ ! -f "${ds_dataroot}/ted-train.csv" ]; then
+    echo "Warning: It looks like you don't have the TED-LIUM corpus "          \
+         "downloaded and preprocessed. Make sure \$ds_dataroot points to the " \
+         "folder where the TED-LIUM data is located, and that you ran the " \
+         "importer script before running this script."
+fi;
```

Comment:
I agree on adding the file name of the importer script here. It'll save us many a random issue filed by contributors.